MapboxSearch/autofill/src/main/java/com/mapbox/search/autofill/AddressAutofillImpl.kt

```diff
@@ -117,6 +117,7 @@ internal class AddressAutofillImpl(
         fun create(
             app: Application,
             locationProvider: LocationProvider?,
+            locationObservationTimeout: Long?,
```

Comment:
Seems weird to have this as apart of the `create` method for the `AddressAutofillImpl` class. The property is a value on a singleton so I don't know why it's associated here.

Reply:
This is the 2nd option (please see my other comment above)